gql/createPost.gql

```diff
@@ -0,0 +1,9 @@
+mutation createPost($title: String!, $url: String!) {
```

Comment:
I'm not sure should we create another folder for gql files. This way can break "component-based" folder structure and also I'm planning to make plugin system on RAN! which basically will be component based. But this way also can be better for cleaner code. What's your opinion?